ddtrace/internal/datastreams/schemas/schema_builder.py

```diff
@@ -16,7 +15,7 @@
 class SchemaBuilder:
     max_depth = 10
     max_properties = 1000
-    CACHE = lru_cache(maxsize=32)
+    CACHE: Dict[str, Schema] = {}
```

Comment:
Why did we change this here?

Reply:
All this schema code is new and was added by me in my last PR, I was using the lru_cache wrong and just switched to using a dict, which should be fine.